server/webapp/WEB-INF/rails.new/app/presenters/api_v1/version_info_representer.rb

```diff
@@ -0,0 +1,50 @@
+##########################################################################
+# Copyright 2015 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module ApiV1
+  class VersionInfoRepresenter < ApiV1::BaseRepresenter
+    def initialize(version_info, system_environment)
+      @version_info = version_info
+      @system_environment = system_environment
+      super(@version_info)
+    end
+
+    link :doc do
+      'http://api.go.cd/#versionInfo'
```

Comment:
@arvindsv — are we going to call this "API"?

We do have a special route for internal API if that is what we're going to call this.